src/mmw/js/src/modeling/views.js

```diff
@@ -199,6 +258,8 @@ var ToolbarTabContentsView = Marionette.CollectionView.extend({
     childView: ToolbarTabContentView,
 
     onRender: function() {
+        // TODO: Move this logic to the template (active tab should
+        // be stored in model state somewhere).
```

Comment:
Yeah `initialScenario` is `activeScenarioSlug`, but I thought the name `initialScenario` made more sense as it got passed down and the view was rendered initially. Open to whatever changes though.